CHANGELOG.md

```diff
@@ -1 +1,99 @@
 # Changelog
+
+Items starting with `DEPRECATE` are important deprecation notices. For more
+information on the list of deprecated flags and APIs please have a look at
+https://docs.docker.com/engine/deprecated/ where target removal dates can also
+be found.
+
+## 18.01.0-ce (2018-01-DD)
+
+### ---Triage---
+
+* Legacy images replacement with multi-arch ones on AArch64 [moby/moby#35773](https://github.com/moby/moby/pull/35773)
+* Vendor: update archive/tar [moby/moby#35728](https://github.com/moby/moby/pull/35728)
+
+### --Delete-- (For review only)
+- Fixed typo DEREPCATED -> DEPRECATED. [moby/moby#35899](https://github.com/moby/moby/pull/35899)
+- Fix typo in link [docker/cli#759](https://github.com/docker/cli/pull/759)
+* Bump API version to 1.36 [moby/moby#35738](https://github.com/moby/moby/pull/35738)
```

Comment:
reverted in https://github.com/docker/docker-ce/pull/378

note that API version bumps should normally be included in the changelog. We can copy the exact changes from the API history; https://github.com/docker/docker-ce/blob/17.12/components/engine/docs/api/version-history.md#v135-api-changes

There's no API changes in 18.01, so not for this release :smile:

Reply:
Everything under the delete header I intended to delete. 
Do you agree with that?

Reply:
Yes, they are all ok to remove (just pointed out the API things for future change logs 😅 )